spec/models/article_spec.rb

```diff
@@ -31,7 +31,7 @@ def build_and_validate_article(*args)
 
     context "when published" do
       before do
-        allow(subject).to receive(:published?).and_return(true) # rubocop:disable RSpec/NamedSubject
+        allow(article).to receive(:published?).and_return(true)
```

Comment:
The inline disabling didn't work, so I just changed this to make the pre-commit hook happy.

Reply:
This is causing spec on line 36 to fail. It's probably because the spec on line 36 isn't using `article` but `subject`.

Reply:
Thanks @maestromac, was about to look into this now, much appreciated 👍